python/ray/tests/test_basic_3.py

```diff
@@ -160,7 +160,7 @@ def f():
         pid, g_id = nested.pop(0)
         ray.get(g_id)
         del g_id
-        wait_for_pid_to_exit(pid)
+        wait_for_pid_to_exit(pid, timeout=900)
```

Comment:
Why is this necessary? It looks like it is hiding some issue.

Reply:
This is for debugging. Feel free to take a look later after @author-action-required is removed.

Reply:
Oh, my bad. I got a notification "@mwtian mwtian requested a review from AmeerHajAli as a code owner 4 days ago" so I thought you forgot to remove it.

Reply:
Ready now.